CMake/External_VXL.cmake

```diff
@@ -111,6 +95,7 @@ ExternalProject_Add(VXL
     -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
     -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
     -DCMAKE_INSTALL_LIBDIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/lib
+    -DCMAKE_PREFIX_PATH:PATH=${fletch_BUILD_INSTALL_PREFIX}
```

Comment:
Agreed. We will still need to add each of the supporting packages we build to the DEPENDS list though, so they are built in the correct order.